lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala

```diff
@@ -647,6 +650,7 @@ private[scalacoptions] trait ScalacOptions {
   val privateOptions: Set[ScalacOption] = ListSet(
     privateNoAdaptedArgs,
     privateKindProjector,
+    privateSafeInit,
```

Comment:
Yes it will

Reply:
I know there is a mention of the word "experimental" but it's not under the `Experimental` section https://docs.scala-lang.org/scala3/reference/other-new-features/safe-initialization.html

Reply:
Also this is check so not sure if the experimental refers to the fact that it wouldn't catch all of them or if it would fail on false positives like the unused imports

Reply:
I don't know why it's not in the experimental section, maybe an oversight 🤷

Reply:
> Yes it will. Should I remove it?

Yes, Seth explained here why: https://github.com/typelevel/scalac-options/pull/85#issuecomment-1788035786

Reply:
Adressed